src/fragmentarium/ui/fragmentariumSearch.ts

```diff
@@ -0,0 +1,12 @@
+export interface SearchGroupParams {
+  number: string | null | undefined
+  id: string | null | undefined
+  title: string | null | undefined
+  pages: string | null | undefined
+  transliteration: string | null | undefined
+}
+
+export interface FragmentariumSearchParams extends SearchGroupParams {
+  fragmentService
+  fragmentSearchService
+}
```

Comment:
Are these props or parameters? The naming suggests that they belong to specific components. If that is the case they should be defined along with the component. If it something more generic it should be named accordingly e.g. `FragmentSearchParameters`.

You could also consider having a full model for the parameters. Then e.g. the transliteration replace could be handled in the model instead of in the components.

Reply:
i justs defined them in the corresponding components and removed this file